src/components/Header/HeaderItem.jsx

```diff
@@ -0,0 +1,50 @@
+import { Box, ButtonBase, Typography, useTheme } from "@mui/material";
```

Comment:
We should separate local and global import for better readability

global comes first then local imports

Reply:
Can you explain a bit more about this?
Do you mean I create a separate file for Importing components related to MUI?